notifications_utils/polygons.py

```diff
@@ -68,6 +70,17 @@ class Polygons():
     output_precision_in_decimal_places = 6
 
     def __init__(self, polygons, utm_crs=None):
+        """
+        This class accepts either:
+        - a list of lists of coordinate pairs defined in WGS84 degrees
+        - a list of lists of coordinate pairs defined in WGS84 degrees,
```

Comment:
How come we need this? As I understand it, a `utm_crs` is only required when we've projected the WGS84 coordinates onto a plane.